.github/workflows/build.yml

```diff
@@ -5,13 +5,40 @@ on:
     branches: [master]
 
 jobs:
-    test:
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v2
-            - name: Set up JDK 11
-              uses: actions/setup-java@v1
-              with:
-                java-version: 11
-            - name: Build with Maven
-              run: mvn --no-transfer-progress package
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build with Maven
+        run: mvn --no-transfer-progress package
+  liquibase:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:12-alpine
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_USER: simplq
+          POSTGRES_DB: simplq
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 30s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - name: Checkout repo code
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Build project
+        run: mvn -pl simplq -q install -DskipTests
+      - name: Run liquibase migration
+        run: mvn -pl simplq -q liquibase:update -D db.username=simplq -D db.url=jdbc:postgresql://localhost:5432/simplq
```

Comment:
We have to add one more step at the end where we run the test?

```
            - name: Run tests
              run: mvn --no-transfer-progress package
```

Reply:
We do not need this step. We have another one which runs the tests

Reply:
Didn't understand. Did you mean the travis CI one?

Reply:
Line 17 @daltonfury42

Reply:
Oh I missed it.

I was thinking, if the integration tests that we have, they ran against the DB post migration, it would catch regressions due to migrations not being written etc. What do you think?

Reply:
For that we will need to write integration tests, but for that we will create another pr.

Reply:
ok, makes sense.